datasets/_externals/ext_gleif.yml

```diff
@@ -15,8 +15,6 @@ deploy:
   disk: "20Gi"
   # premium: true
 load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
-coverage:
```

Comment:
Why are we getting rid of this?

Reply:
duplication https://github.com/opensanctions/opensanctions/pull/458/files/c41bff52b52485267cd62712957922b8d47d0fe2#diff-4a36d4536b29d23bd921ee69a2f61e304cd1d562208442703359fd54ed5d301bL7

Reply:
me moron. me go home

Reply:
better safe than sorry!